buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/RunTask.java

```diff
@@ -36,12 +39,19 @@ public void beforeStart() {
         int debugPort = 8000;
         int httpPort = 9200;
         int transportPort = 9300;
+        Map<String, String> additionalSettings = System.getProperties().entrySet().stream()
```

Comment:
We need to track this stuff as an input for incremental build and caching to work as expected. I'd move all this into a `getAdditionalSettings()` method annotated with `@Input`.